servicemanager/servicemanager.go

```diff
@@ -134,9 +204,35 @@ func (sm *ServiceManager) InstallService(
 	)
 }
 
+// RemoveService removes service from the system.
+func (sm *ServiceManager) RemoveService(serviceID string) (err error) {
+	return <-sm.actionHandler.Execute(serviceID,
+		func(id string) error {
+			return sm.doRemoveService(serviceID)
+		},
+	)
+}
+
+// AllocateLayersSpace allocate space by removing outdated services.
+func (sm *ServiceManager) AllocateLayersSpace(extraSpace int64) (allocatedLayersSize int64, err error) {
+	_, allocatedLayersSize, err = sm.removePendingServices(0, extraSpace)
```

Comment:
Combine in one line. Or remove named return value.

Reply:
Done